community/postsrsd/APKBUILD

```diff
@@ -0,0 +1,49 @@
+# Contributor: Kevin Daudt <ops@ikke.info>
+# Maintainer: Kevin Daudt <ops@ikke.info>
+pkgname=postsrsd
+pkgver=1.4
+pkgrel=0
+pkgdesc="Postfix Sender Rewriting Scheme daemon"
+url="https://github.com/roehling/postsrsd"
+arch="all"
+license="GPL2"
+depends=""
+makedepends="cmake help2man"
+pkgusers="postsrsd"
+install="$pkgname.pre-install $pkgname.post-install"
+subpackages="$pkgname-doc"
+source="$pkgname-$pkgver.tar.gz::https://github.com/roehling/postsrsd/archive/$pkgver.tar.gz
+	10-fix-defaults.patch
+	postsrsd.init"
+builddir="$srcdir/$pkgname-$pkgver"
+
+build() {
+	cd "$builddir"
+
+	mkdir build
+	cd build
+
+	cmake .. -DCMAKE_INSTALL_PREFIX=/usr/ -DCMAKE_BUILD_TYPE=Release -DGENERATE_SRS_SECRET=OFF -DCONFIG_DIR=/etc/postsrsd
```

Comment:
Please try to break lines at 80 chars. For readability you could also break after each cmake options.
And always return 1 if its not the last command in a function (we do not set -e).